RELEASING.md

```diff
@@ -31,4 +31,10 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 
 ## Releasing
 
-The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
+The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [maintainers](MAINTAINERS.md).
+
+1. Create a tag, e.g. v2.1.0, and push it to the GitHub repo.
```

Comment:
*Nit*: Fix the numberings?

Reply:
Its in markdown. If you click on view file you will see the numbering auto corrected. https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
My bad 🤦🏼‍♀️